model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -342,6 +342,39 @@ def copy_graph_with_ops(graph: Graph) -> Graph:
 
     # Nodes postprocessing stage in new graph
     for op in new_graph.get_op_nodes():
+        # tensor names restoring
```

Comment:
I want to see a unit test for this functionality. We have a unit test for IE Engine and example of the IR `mo/utils/unittest/test_data/mxnet_synthetic_gru_bidirectional_FP16_1_v6.xml`. We should update the XML with information about port names for some of the operations and verify that they are parsed correctly in the unit test.

Reply:
We also can modify output ports for Const node to make sure that both old and new style of out port numbering works.

Reply:
Test were updated. I try to modify output ports in other place, but it is hard and needs some refactoring in MO IR reader and in IR Engine. I think that it is the best decision now.